fixtures/flows/briefResponse/atm.js

```diff
@@ -1,30 +1,33 @@
-export const respond = async (params) => {
-    console.log('Starting to respond to atm brief');
+import * as util from '../../flows/utils'
 
-    await clickSubmitApplication();
-    await matchText('a', 'Enter a date for when you can start the project');
-    if (params.criterias) {
-        for (let i = 0; i < params.criterias.length; i++) {
-            console.log(params.criterias[i])
-            await matchText('a', params.criterias[i].criteria);
-        }
-    }
-    await matchText('a', 'You must upload your written proposal');
-    await matchText('a', 'You must add a phone number');
-
-    await typeInReactInput('availability', { numberOfCharacters: 100 });
-    if (params.criterias) {
-        for (let i = 0; i < params.criterias.length; i++) {
-            await typeInReactInput(`criteria.${i}`, { numberOfWords: 500 });
-        }
-    }
-    await upload('file_0', 'document.pdf');
-    await typeInReactInput('respondToPhone', { value: '0123456789' });
+/* eslint-disable no-await-in-loop */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`